source/structure.html

```diff
@@ -0,0 +1,628 @@
+                                 alt="Rasmus Lerdorf">
+                            <div class="leading-6 space-y-1">
+                                <div class="text-lg font-bold text-black">Rasmus Lerdorf</div>
+                                <div class="text-[#7f52ff]">Administration</div>
+                                <ul class="flex space-x-4 list-none p-0 m-0">
+                                    <li>
+                                        <a href="https://twitter.com/rasmus" class="text-gray-400 hover:text-gray-500" rel="noreferrer noopener" target="_blank">
+                                            <span class="sr-only">Twitter</span>
+                                            <svg class="w-5 h-5" fill="currentColor" viewBox="0 0 20 20" aria-hidden="true">
+                                                <path d="M6.29 18.251c7.547 0 11.675-6.253 11.675-11.675 0-.178 0-.355-.012-.53A8.348 8.348 0 0020 3.92a8.19 8.19 0 01-2.357.646 4.118 4.118 0 001.804-2.27 8.224 8.224 0 01-2.605.996 4.107 4.107 0 00-6.993 3.743 11.65 11.65 0 01-8.457-4.287 4.106 4.106 0 001.27 5.477A4.073 4.073 0 01.8 7.713v.052a4.105 4.105 0 003.292 4.022 4.095 4.095 0 01-1.853.07 4.108 4.108 0 003.834 2.85A8.233 8.233 0 010 16.407a11.616 11.616 0 006.29 1.84" />
+                                            </svg>
+                                        </a>
+                                    </li>
+                                    <li>
+                                        <a href="https://github.com/rlerdorf" class="text-gray-400 hover:text-gray-500" rel="noreferrer noopener" target="_blank">
+                                            <span class="sr-only">GitHub</span>
+                                            <svg class="w-5 h-5" fill="currentColor" viewBox="0 0 16 16" aria-hidden="true">
+                                                <path d="M8 0C3.58 0 0 3.58 0 8C0 11.54 2.29 14.53 5.47 15.59C5.87 15.66 6.02 15.42 6.02 15.21C6.02 15.02 6.01 14.39 6.01 13.72C4 14.09 3.48 13.23 3.32 12.78C3.23 12.55 2.84 11.84 2.5 11.65C2.22 11.5 1.82 11.13 2.49 11.12C3.12 11.11 3.57 11.7 3.72 11.94C4.44 13.15 5.59 12.81 6.05 12.6C6.12 12.08 6.33 11.73 6.56 11.53C4.78 11.33 2.92 10.64 2.92 7.58C2.92 6.71 3.23 5.99 3.74 5.43C3.66 5.23 3.38 4.41 3.82 3.31C3.82 3.31 4.49 3.1 6.02 4.13C6.66 3.95 7.34 3.86 8.02 3.86C8.7 3.86 9.38 3.95 10.02 4.13C11.55 3.09 12.22 3.31 12.22 3.31C12.66 4.41 12.38 5.23 12.3 5.43C12.81 5.99 13.12 6.7 13.12 7.58C13.12 10.65 11.25 11.33 9.47 11.53C9.76 11.78 10.01 12.26 10.01 13.01C10.01 14.08 10 14.94 10 15.21C10 15.42 10.15 15.67 10.55 15.59C13.71 14.53 16 11.53 16 8C16 3.58 12.42 0 8 0Z"></path>
+                                            </svg>
+                                        </a>
+                                    </li>
+                                </ul>
+                            </div>
+                        </div>
```

Comment:
Neither Dmitry nor Rasmus have been participating in the admin; are we including them because of their involvement with initial founding?

Reply:
Dmitry participated in a couple of first meetings.

@dstogov, @rlerdorf, folks would you like to stay on this list?

Reply:
@saundefined Let's move Dmitry and Rasmus to the Community section as PHP Fellow for now.

Reply:
Done, demo updated